pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -106,7 +106,7 @@ symlinkJoin {
 
   '' + (lib.optionalString (stdenv.targetPlatform.isDarwin && !isGhcjs && !stdenv.targetPlatform.isiOS) ''
     # Work around a linker limit in macOS Sierra (see generic-builder.nix):
-    local packageConfDir="$out/lib/${ghc.name}/package.conf.d";
+    local packageConfDir="${libDir}/package.conf.d";
```

Comment:
Shouldn't this be `${packageCfgDir}` while we're at it?

Reply:
Yes, indeed. I can update that too. The semantic is the same.